src/i18n/strings/hu.json

```diff
@@ -704,6 +704,13 @@
     "Missing media permissions, click the button below to request.": "Hiányzó média jogosultságok, kattints a gombra alul a jogok megadásához.",
     "Request media permissions": "Média jogosultságok megkérése",
     "Voice & Video": "Hang és videó",
+    "Voice settings": "Hangbeállítások",
+    "Video settings": "Videóbeállítások",
+    "Voice processing": "Hangfeldolgozás",
+    "Automatic gain control": "Automatikus erősségszabályozás",
+    "Echo cancellation": "Visszhangcsökkentés",
+    "Noise suppression": "Zajcsökkentés",
+    "Connection": "Kapcsolat",
```

Comment:
Please don't edit locales except `en_EN.json`; you should just run `yarn i18n` - translations are handled through https://translate.element.io/